chia/_tests/blockchain/test_augmented_chain.py

```diff
@@ -93,7 +93,7 @@ async def test_augmented_chain(default_10000_blocks: list[FullBlock]) -> None:
     with pytest.raises(KeyError):
         await abc.prev_block_hash([blocks[2].header_hash])
 
-    with pytest.raises(ValueError, match="Err.GENERATOR_REF_HAS_NO_GENERATOR"):
+    with pytest.raises(ValueError, match="Err\\.GENERATOR_REF_HAS_NO_GENERATOR"):
```

Comment:
etc...  in various files.


```suggestion
    with pytest.raises(ValueError, match=re.escape(str(Err.GENERATOR_REF_HAS_NO_GENERATOR))):
```

Reply:
Went with a `Err.CODE.name` instead of `str(Err.CODE)` but otherwise taken